html_css/intermediate_html/form-validations.md

```diff
@@ -186,9 +186,9 @@ It's also worth noting client-side validations are not a silver bullet for ensur
 <div class="lesson-content__panel" markdown="1">
 
 1. Read and follow along to [MDN's Client-Side Form Validation Guide](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation)
-2. SitePoint's [Complete Guide to HTML Forms and Constraint Validation Guide](https://www.sitepoint.com/html-forms-constraint-validation-complete-guide/)
```

Comment:
I think this could either stay here as an assignment item, if only for the CSS Validation Styling section in the article, or it could make a good additional resource. It does cover some of the same stuff as the MDN article albeit slightly differently, and again the CSS validation styling I think could be useful for users to read about.

Reply:
I've added it back to the assignment section